winit/Cargo.toml

```diff
@@ -0,0 +1,122 @@
+[package]
+authors = ["The winit contributors", "Pierre Krieger <pierre.krieger1708@gmail.com>"]
```

Comment:
Nit: We should IMO just remove this, the `authors` field isn't required on crates.io nowadays anyhow, and "The winit contributors" also includes Pierre.
```suggestion
```